src/acl/acl.ts

```diff
@@ -142,20 +142,17 @@ export function internal_getContainerPath(resourcePath: string): string {
 }
 
 /**
- * Verify whether an ACL was found for the given Resource.
+ * Verify whether the given Resource has a resource ACL attached.
  *
- * A Resource fetched with its ACL (e.g. using [[getSolidDatasetWithAcl]]) _might_ have a resource ACL attached, but
- * we cannot be sure: it might be that none exists for this specific Resource (in which case the
- * fallback ACL applies), or the currently authenticated user (if any) might not have Control access
- * to the fetched Resource.
+ * A Resource returned using [[getSolidDatasetWithAcl]], i.e., the returned `dataset,
+ * has its resource ACL attached if its resource ACL exists and the current user
+ * has Control access to the Resource. Otherwise, the Resource will not have a resource ACL attached.
  *
- * This function verifies that the Resource's ACL is accessible.
- *
- * Please note that the Web Access Control specification is not yet finalised, and hence, this
- * function is still experimental and can change in a non-major release.
+ * _Note_: The Web Access Control specification is not yet finalised. As such, this
+ * function is still experimental and subject to change.
  *
  * @param resource A Resource that might have an ACL attached.
- * @returns Whether `dataset` has an ACL attached.
+ * @returns Whether the Resource has a resource ACL attached.
```

Comment:
So, I kind of wanted to update to  @returns `true` if the Resource has a ...
because the function states 
`
return ( resourceAcl !== null && ... )  
`
but I got a little confused because we put the following
`
: resource is Resource & WithResourceAcl & WithAccessibleAcl
`, so that typedoc states it returns a Resource  followed by the @returns sentence.

Reply:
This type signature is a Typescript type guard, so it does return a boolean, and this boolean is true if the `resource` argument had properties for the types `Resource`, `WithResourceAcl`, `WithAccessibleAcl`. The `@return` could be `true if the Resource has a resource ACL attached`.

Reply:
Yup -- 
but the typedoc outputs the following markdown

```
**Returns:** *resource is Resource & WithFallbackAcl*

Whether the Resource has a fallback ACL attached.

```

So, I'll update @returns to `true if ...`

but it'll output

```
**Returns:** *resource is Resource & WithFallbackAcl*

`true` if the Resource ...

```

I wasn't sure if there was a way that typedoc didn't output the first part.